corehq/util/context_processors.py

```diff
@@ -176,6 +176,7 @@ def enterprise_mode(request):
     return {
         'enterprise_mode': settings.ENTERPRISE_MODE,
         'is_saas_environment': settings.IS_SAAS_ENVIRONMENT,
+        'is_dimagi_environment': settings.IS_DIMAGI_ENVIRONMENT,
```

Comment:
What's the difference between `IS_SAAS_ENVIRONMENT` and `IS_DIMAGI_ENVIRONMENT`?

Reply:
I suppose what I want to know is if whether or not a "SaaS environment" refers to "a dimagi SaaS env" or not. If so, one could drop `IS_DIMAGI_ENVIRONMENT`.

Reply:
That is a good question. Both settings would be true for a Dimagi environment, however I don't know if they can be used interchangably. Since `IS_SAAS_ENVIRONMENT` is true if the environment is set to `production` or `staging`, self-hosters could potentially set their environment to production and then the variable would be true. They can easily also set `IS_DIMAGI_ENVIRONMENT` to true, however I think this setting is at least more explicit in pointing out that it's not just a general SaaS environment, but something specific to Dimagi.

@kaapstorm I'm curious to hear if you perhaps have any thoughts regarding the above?

Reply:
I'm not sure. I've asked about, and I'll let you know.

Reply:
Apparently SaaS environments are a subset of Dimagi environments. (Does india fall under SaaS or ID/Solutions?)

Reply:
Ah, thanks for the clarification @kaapstorm! I admit this naming can come across as a little confusing and wonder if something like `IS_DIMAGI_SAAS_ENVIRONMENT` wouldn't have been a better choice. Either way, given that it's only a subset I'll keep to using `IS_DIMAGI_ENVIRONMENT`.